shaptools/netweaver.py

```diff
@@ -243,7 +243,7 @@ def install(
                 cwd=' SAPINST_CWD={}'.format(cwd) if cwd else '')
         result = shell.execute_cmd(cmd, root_user, password, remote_host)
         if result.returncode and raise_exception:
-            raise NetweaverError('SAP Netweaver installation failed')
+            raise NetweaverError('SAP Netweaver installation failed. Logs are saved into installation_folder, default: \"/tmp/swpm_unattended/\" (sapinst_dev.log and sapinst.log)')
```

Comment:
The `installation_folder` does not exist in shaptools. May be just write something like: " Please check swpm installation logs (sapinst_dev.log and sapinst.log) located at swpm installation folder for further information. " 
You can mention the default location (  \"/tmp/swpm_unattended/\" )  there maybe

Reply:
I was actually thinking that we should pass the `installation_folder` as parameter to the function so we can print the exact location. 
So from state we get the pillar and pass to this function.

I think in this case the exact `location` is really important for improving the UX.

Reply:
@Simranpal  Why do you say that `/tmp/swpm_unattended` is the default location?
This value is coming from other pieces of cose (salt formula in our case).
@MalloZup I think you are looking for `cwd` parameter. The installation stores the files in that folder (and if if's not set in a predefined folder defined by SAP, that I don't remember now the structure).

In any case, our deployments sets the `cwd` parameter to `/tmp/swpm_unattended` by default

Reply:
ok thx @arbulu89  I was thinking also , can we print the `error_msg` of the installer?

something like this in pseudo_code
`Netweaver Install fail. Error: {0}  Check logs at folder{1}".format(error_msg, cwd)`

Reply:
@arbulu89 my comment can be improved, I meant to add there"by default"

Reply:
updated thx